### DISCUSSION TITLE
Print stops after Start Gcode

### DISCUSSION THREAD

**Discussion by devynb22** [Jyers/Marlin#1272] [Q&A] [answered]
I compiled the firmware using the "UBL-NoProbe-3x3" template. I have an all metal hotend and a bmg clone so I made the necessary changes within that config file. 

Now my printer stops printing immediately after completing the start Gcode. I did change the start Gcode to include "M420 S1" but did not change much else about my Start Gcode. I tested prints that were successful before the firmware swap and those exhibit the same issue. Thank you for any help.

Start Gcode:

G28 ; Home all axes
M420 S1 ; Load Mesh file
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little
G1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z1.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
; End of Start Gcode

**Comment by adam3654** [comment, marked as answer]
Are you telling the printer to heat up the bed and hotend?

**Reply by devynb22** [reply]
Is that not controlled by the slicer export? If not then would I just need to use different hotend and bed temperature values in the start gcodes for different material presets? (BTW: Using Ideamaker slicer)

**Reply by devynb22** [reply]
In Ideamaker I have "Insert Heatup GCode Automatically in Start GCode" enabled with the bed heatup first. The bed and nozzle heat up fine, and the lines print as well, but then the printer parks itself at where it stopped printing the line and the display says "100%" and "Confirm."

**Reply by adam3654** [reply]
I only have experience with cura. After slicing and saving the code file, open it in a text editor and see what it's doing.

**Reply by devynb22** [reply]
Thank you for the tip, I forgot to even check the GCode text file. Ideamaker exports with "M1001" after the start GCode and "M1002" near the end. I removed them from the source code and now the print actually starts. The print is still running but I think it works now.

**Reply by adam3654** [reply]
Check this thread out, looks like you can tell the slicer not to add those lines

http://forum.raise3d.com/viewtopic.php?t=4610
